### ISSUE TITLE
2 or 3 different accounts by social network

### ISSUE THREAD

**Issue by kodjoe** [pavelk2/social-feed#338]
Hi can we provide 2 or 3 different accounts by social network. 
I mean for example if we use instagram, can we have 2 different accounts in the feed.  
By modifying the original code : 

    // INSTAGRAM
    instagram:{
        accounts: ['&24************47'],
		//Array: Specify a list of accounts from which to pull posts                             
        client_id: '64************4d',       
		//String: Instagram client id (option if using access token)
        access_token: '24************rf', 
		//String: Instagram access token
        limit: 20    
    },	

By this code : 
		//accounts: ['&24************47','&14************23'],  
		//Array: Specify a list of accounts from which to pull posts   
        //client_id: ['64************4d','c3************40'],       
		//String: Instagram client id (option if using access token)
        //access_token: ['24************rf','14************9a'],   
		//String: Instagram access token
        limit: 20    
		//Integer: max number of posts to load
    },

thank you

**Comment by HartLarsson** [comment]
because Instagram like also Fb limit to your account and your Token the abilty to grab feed is impossible (i think) to grabs with same token from different accounts.

What you suggest is an athorization extension to multiple feed/accounts and need a complete plugin rework.

**Comment by kodjoe** [comment]
Yep i have 2 differrent instagram accounts, it means 2 client id, 2 token and 2 different user id. My code example stream only one account. And if i put 2 user id with same cliend id and token, only the account of the second user id will work. So to you, it's impossible to mix them and see them in the same wall with this code ? Is it hard to achieve that ???

Thanks a lot for your answer

**Comment by HartLarsson** [comment]
As i told before need a complete core code rewrite.

**Comment by kodjoe** [comment]
ok thanks a lot for your answer @HartLarsson ;D so we will stay with one account by social network at this time !!!
